grails-app/controllers/org/pih/warehouse/requisition/RequisitionTemplateController.groovy

```diff
@@ -86,6 +86,79 @@ class RequisitionTemplateController {
         }
     }
 
+    def exportPage = {
+        render(view:"exportPage")
+    }
+
+    def exportAsCsv = {
+        def requisitionItems = []
+        def origins = []
+        def destinations = []
+
+        params.origin.each { origin ->
+            origins << Location.get(origin)
+        }
+        params.destination.each { destination ->
+            destinations << Location.get(destination)
+        }
+
+        def requisitions = requisitionService.getRequisitionTemplates(origins, destinations)
+        requisitionService.getRequisitionTemplatesItems(requisitions).groupBy { it.product }.each { product, value ->
+            requisitionItems << [
+                    product: product,
+                    productCode: product.productCode ?: "",
+                    productName: product.name ?: "",
+                    category: product?.category?.name ?: "",
+                    pricePerUnit: product?.pricePerUnit ?: "",
+            ]
+        }
+
+        def sw = new StringWriter()
+
+        try {
+            if (requisitionItems) {
+                sw.append("Item code").append(",")
```

Comment:
Use the localized message. Something like this, but search the code to make sure that's the right way to do it within a controller.
```
sw.append(g.message(code: 'product.productCode.label')).append(",")
```